tmk_core/common/eeconfig.c

```diff
@@ -49,6 +49,7 @@ void eeconfig_init_quantum(void) {
   eeprom_update_byte(EECONFIG_VELOCIKEY,      0);
 #ifdef EECONFIG_RGB_MATRIX
```

Comment:
This ifdef should be removed, I think. Otherwise the rgbmatrix struct could be filled with `0xFF` in the event of the EEPROM being reset, then RGB Matrix being enabled.

Reply:
good point, fixed